index.html

```diff
@@ -770,8 +771,8 @@
 
 reader.addEventListener("reading", event => {
   for (let record of event.message.records) {
-    let article =/[aeio]/.test(record.data().title) ? "an" : "a";
-    console.log(`${record.data().name} is ${article} ${record.data().title}`);
+    let article =/[aeio]/.test(record.toJSON().title) ? "an" : "a";
```

Comment:
I would factor it out here

const json = record.toJSON();

Reply:
You are generating a new object each time...

Reply:
Thanks, done.